packages/turf/package.json

```diff
@@ -64,14 +64,14 @@
     "test": "echo '@turf/turf tests run in the last-checks step'"
   },
   "devDependencies": {
+    "@rollup/plugin-buble": "^0.21.3",
```

Comment:
Are we actually using `@rollup/plugin-buble` anywhere... I couldn't find any use of it?

Reply:
On interesting, I thought we were using it in @turf/turf's special rollup but I guess not. Its on my todo list to swap to babel anyhow, since babel is roughly 60x more popular. That would help fix other issues related to es6 syntax creeping in.

Reply:
I'm happy for it to sit there unused and it'll eventually go with the switch to babel.